nilearn/decoding/space_net.py

```diff
@@ -1134,6 +1131,27 @@ def _binarize_y(self, y):
         self.n_classes_ = len(self.classes_)
         return y
 
+    def score(self, X, y, sample_weight=None):
+        """Returns the mean accuracy on the given test data and labels.
+
+        Parameters
+        ----------
+        X : list of Niimg-like objects
+            See http://nilearn.github.io/building_blocks/
```

Comment:
Actually the right URL is http://nilearn.github.io/manipulating_visualizing/manipulating_images.html#niimg 

About having shorter URL, What about an URL shortener like `git.io` ? http://git.io/v4fU8
I agree that it is not very intuitive ...